src/reporter.ts

```diff
@@ -184,13 +240,22 @@ class RPReporter implements Reporter {
     }
   }
 
+  findLaunchLog(log: LogRQ): LogRQ {
+    for (const [key, value] of this.launchLogs) {
```

Comment:
What the reason iterate here every time if `launchLog` is a `Map` with keys = `log.message`?
Looks like `findLaunchLog` is redundant.